### PR TITLE
Create ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-54749](https://cumulocity.atlassian.net/browse/MTM-54749)
Original PR: [7286](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7286)

[MTM-54749]: https://cumulocity.atlassian.net/browse/MTM-54749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Allow possibility for global asset search when duplicating private smartrule but update context upon duplication (#7188) [GRAFT][release/cd] (#7286)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md]: The Claude AI suggested to use the following title:
```suggestion
title: Allow global asset search when duplicating private smart rule and update context
```

File: content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-54749
+version: 1020.33.0
+---
+Allow possibility for global asset search when duplicating private smartrule but update context upon duplication (#7188) [GRAFT][release/cd] (#7286)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md]: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT application "Cockpit", it is possible to duplicate smart rules to reuse their logic for other assets. Previously, when duplicating a private smart rule, the asset search was limited to the current context of the smart rule. With this change, the asset search is now enabled globally when duplicating a private smart rule, allowing users to select any asset for the duplicated smart rule. However, upon duplication, the context of the duplicated smart rule is updated to match the newly selected asset. This change improves the usability and flexibility when reusing smart rule logic across different assets in Cockpit.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by iiar-c8y** [comment]
Unfortunately release note was not picked up from squash commit message, will update.

File: content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md (hunk 2 above)
Comment (iiar-c8y) [inline content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md:17]: When creating a duplicate of the local smart rule, it is now possible to select asset from outside of hierarchy. Duplicate of the rule is now created within the context of the selected asset.

Example:
Duplicate a rule form one group to another and enable it for all child assets would require following steps:
1. Select local smart rule to create duplicate from.
2. When updating configuration click search for target asset and find target group.
3. Ensure that checkbox 'Activate also for X assets' selected if you want to enable the rule for children of selected asset.
4. Save new rule
5. Check alert message displayed which should contain a link to target group. Navigate to new location using it to discover newly created rule.

Comment (iiar-c8y) [inline content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md]:
```suggestion
When creating a duplicate of the local smart rule, it is now possible to select asset from outside of hierarchy. Duplicate of the rule is now created within the context of the selected asset.

Example:
Duplicate a rule form one group to another and enable it for all child assets would require following steps:
1. Select local smart rule to create duplicate from.
2. When updating configuration click search for target asset and find target group.
3. Ensure that checkbox 'Activate also for X assets' selected if you want to enable the rule for children of selected asset.
4. Save new rule
5. Check alert message displayed which should contain a link to target group. Navigate to new location using it to discover newly created rule.
```

File: content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md (hunk 1 above)
Comment (iiar-c8y) [inline content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md]:
```suggestion
title: Allow global asset search when duplicating private smart rule and update context for new rule
```

File: content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-54749
+version: 1020.33.0
+---
+When creating a duplicate of the local smart rule, it is now possible to select asset from outside of hierarchy. Duplicate of the rule is now created within the context of the selected asset.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md]:
```suggestion
When creating a duplicate of a local smart rule, it is now possible to select an asset from outside of the hierarchy. A duplicate of the rule is now created in the context of the selected asset.
```

File: content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+Example:
+Duplicate a rule form one group to another and enable it for all child assets would require following steps:
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md]: Please move these details into the user documentation and just link to it from here.

File: content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+date: ""
+title: Allow global asset search when duplicating private smart rule and update context for new rule
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md]:
```suggestion
title: Allow global asset search when duplicating a local smart rule
```

File: content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md (hunk 4 above)
Comment (iiar-c8y) [reply content/change-logs/application-enablement/ui-c8y-1020-33-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md]: Rather would skip this part. Public documentation not really needs this. Customers were complaining for this scenario not working for them, so now it should be resoled. Originally was added to help AI compose a nice message ;)

**Review by iiar-c8y — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@rahultsag Please review and approve.

**Review by rahulc8y — APPROVED** [review]

**Pull request merged**